performance-tracking/src/test/java/com/rakuten/tech/mobile/perf/runtime/internal/AnalyticsBroadcastSpec.java

```diff
@@ -98,6 +99,14 @@ public void shouldBroadcastValidNameAndData() {
     assertThat(analytics.isUrlBlacklisted("rat.rakuten.co.jp")).isFalse(); // missing schema
   }
 
+  @Test public void shouldDisableSendingEvents() {
```

Comment:
this name is a bit weird, maybe `should not send events when disabled` or something would sound better?

Reply:
Yeah, it doesn't really have the context, so that's better.